controllers/dscinitialization/utils.go

```diff
@@ -100,6 +102,14 @@ func (r *DSCInitializationReconciler) createOdhNamespace(ctx context.Context, ds
 				r.Log.Error(err, "Unable to fetch monitoring namespace", "name", monitoringName)
 				return err
 			}
+		} else { // force to patch monitoring namespace with label for cluster-monitoring
+			r.Log.Info("Patching monitoring namespace for Managed cluster", "name", monitoringName)
```

Comment:
I guess such a code duplication fits the general style, doesn't it?

Reply:
it is mostly backport for the same code from downstream to odh.
and to refine the code, we can do have another PR to follow it up.
as for duplication, for ODH itself these labels may not be needed at all so the logic  is only needed to adopt for downstream.